css-positioning/fixed.css

```diff
@@ -0,0 +1,14 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  top: 50px;
+  left: calc(100% / 3);
+  right: calc(100% / 3);
+}
```

Comment:
If the text is aligned center, do you need to be 33% pushed from the left or right? What if screen size changes and the text can't fit on a smaller screen?

Instead we should just make this element span all the way from the left to the right starting points of its parent.